package maid.proto;

message ControllerMeta
{
    optional string service_name = 1;
    optional string method_name = 2;
    optional int32 transmit_id = 3;
    optional bool stub = 4;
    optional bool is_canceled = 5;
    optional bool failed = 6;
    optional string error_text = 7;
    optional bool wide = 8;
    optional int32 depth = 9; // effect when wide is true.
    optional int32 fd = 10;
}
